### PR TITLE
[Core][MPI]  Update test suite for DistributedVectorExporter

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

This PR updates the test suite for the `DistributedVectorExporter` in the file `test_distributed_exporter.cpp`. The change modifies the test case's suite from `KratosCoreFastSuite` to `KratosMPICoreFastSuite`. 

**NOTE:** The test is failing, like the suite was wrong it was not running in the CI, so we lost track when stopped working, @RiccardoRossi took a look

**🆕 Changelog**

- [Update suite](https://github.com/KratosMultiphysics/Kratos/commit/cc52c7450ae0822ce732478836343e9ff946d29b)

### CODE REVIEW

**Comment by loumalouomega** [comment]
You can see the false positives @jcotela : https://github.com/KratosMultiphysics/Kratos/actions/runs/5055290302/jobs/9071307435

~~~sh
 |  /           |                  
 ' /   __| _` | __|  _ \   __|    
 . \  |   (   | |   (   |\__ \  
_|\_\_|  \__,_|\__|\___/ ____/
           Multi-Physics 9.3."1"--2eaf8a4-Custom-x86_64
           Compiled for GNU/Linux and Python3.10 with Clang-14.0
Compiled with threading and MPI support.
Maximum number of threads: 1.
MPI world size:         2.
Process Id: 3883 
TestAssignMPIUniqueModelPartCollectionTagUtility                        OK.
TestBenchmarkDistributedGraphConstructionMPI                            OK.
...blah blah
TestDistributedVectorExporter                                           FAILED!
...blah blah
Ran 236 of 1544 test cases in 0.191161s (2 skipped). 1 failed:
    TestDistributedVectorExporter Failed with messages: 
From rank 0:
Test was reported as successful on this rank, but failed on a different rank.
From rank 1:
Error: Check failed because x[x.GetNumbering().LocalId(total_size-1)] = 20 is not near to 15.0 = 15 within the tolerance 1e-14
in kratos/mpi/tests/cpp_tests/sources/test_distributed_exporter.cpp:60: virtual void Testing::TestDistributedVectorExporter::TestFunction()
~~~

**Comment by loumalouomega** [comment]
After the merge with master should break

**Comment by loumalouomega** [comment]
@RiccardoRossi you can see that with more than 2 partitions the test fails

**Comment by loumalouomega** [comment]
@RiccardoRossi check this

**Comment by loumalouomega** [comment]
Reping @RiccardoRossi

**Comment by RiccardoRossi** [comment]
i have no time to check this now... can u try to debug what happens here?

**Comment by loumalouomega** [comment]
> i have no time to check this now... can u try to debug what happens here?

I already tried and I could find what was the problem. I don't have the time neither for more debug.

File: kratos/mpi/tests/cpp_tests/containers/test_distributed_exporter.cpp [hunk 1]
```diff
@@ -51,10 +51,10 @@ KRATOS_DISTRIBUTED_TEST_CASE_IN_SUITE(DistributedVectorExporter, KratosMPICoreFa
     if(x.GetNumbering().IsLocal(0))
```
Comment (RiccardoRossi) [inline kratos/mpi/tests/cpp_tests/containers/test_distributed_exporter.cpp:52]: I took a quick look into this, and my impression is that x is not initialized ... so it addes the 5,9,15 to a vector that may contain anything...

**Comment by RiccardoRossi** [comment]
@loumalouomega it looks like this fixes it (pending all the tests are finished)

**Review by RiccardoRossi — APPROVED** [review]

**Comment by RiccardoRossi** [comment]
@loumalouomega you opened this, so I can approve it. Feel free to merge it when you like

**Pull request merged**